src/Tuura/Concept/Circuit/Derived.hs

```diff
@@ -199,3 +201,14 @@ internals ints = interfaceConcept $ \s ->
                  if s `elem` ints
                  then Internal
                  else Unused
+
+combinationalGate :: Eq a => Expr a -> a -> CircuitConcept a
```

Comment:
Ideally I'd like that `complexGate (And a b) (And (Not a) (Not b)) c` is the same as `cElement a b c` :)